app/assets/stylesheets/govuk-component/_breadcrumbs.scss

```diff
@@ -15,4 +15,32 @@
     color: $secondary-text-colour;
     text-decoration: none;
   }
+
+  @include media-down(mobile) {
+    &.collapse-on-mobile li {
+      display: none;
+
+      &.parent-taxon {
+        background-image: none;
+        display: block;
+        margin-left: 0;
+        padding-left: 14px;
+        position: relative;
+
+        &:before {
+          border-bottom: 5px solid transparent;
```

Comment:
Required to draw the back arrow in CSS - the prototype lifts the styles from here: http://govuk-elements.herokuapp.com/typography/#typography-links, I've just lifted them from the prototype.